hrjobcontract/tests/phpunit/CRM/Hrjobcontract/BAO/HRJobContractTest.php

```diff
@@ -121,4 +121,30 @@ public function testGetActiveContractsCanReturnContractsActiveOnASpecificPeriod(
     $this->assertCount(2, $activeContracts);
   }
 
+
+  public function testGetCurrentContract() {
+    $contactParams = array("first_name" => "chrollo", "last_name" => "lucilfer");
+    $contactID =  $this->createContact($contactParams);
+    $params = array('contact_id'=> $contactID,
+      'position' => 'spiders boss',
+      'title' => 'spiders boss',
+      'start_date' => '2015-06-01',
+      'end_date' => '2015-10-01');
+    $this->createContract($params);
+    $params = array('contact_id'=> $contactID,
+      'position' => 'spiders boss2',
+      'title' => 'spiders boss2',
+      'start_date' => '2016-02-01',
+      'end_date' => null);
+    $this->createContract($params);
+    $params = array('contact_id'=> $contactID,
+      'position' => 'spiders boss3',
+      'title' => 'spiders boss3',
+      'start_date' => '2014-06-01',
+      'end_date' => '2014-10-01');
+    $this->createContract($params);
+    $current_contract = CRM_Hrjobcontract_BAO_HRJobContract::getCurrentContract($contactID);
+    $this->assertEquals('spiders boss2', $current_contract->title);
```

Comment:
Maybe add some spaces here to improve readability? Also, what about formatting the arrays like:
`
$params = array(
   'field1' => $value,
   'field2' => $value,
   ...
)
`